app/views/albums/show.html.erb

```diff
@@ -0,0 +1,44 @@
+<% @title =  "Edit Album" %>
+
+<h1>Edit Album</h1> 
+
+<%= form_for @album do |a| %>
+  <% if @album.errors.full_messages.present? %>
+    <ul>
+      <% @album.errors.full_messages.each do |message| %>
+        <li>
+          <%= message %>
+        </li>
+      <% end %>
+    </ul>
+  <% end %>
+  <div>
+    <%= a.label :title %>
+    <%= a.text_field :title %>
+  </div>
+  <div>
+    <%= a.label :artist %>
+    <%= a.text_field :artist %>
```

Comment:
Seems you've duplicated this form between `new` & `edit`. In the future, try to make a partial out of this and pass it a local variable which will be the object for `form_for`